### PR TITLE
[Playwright] Set up CI cron job to run playwright tests against Test env Mon-Fri

### PR DESCRIPTION
**Pull request by aweng98**
Created new `nightly-playwright-test` pipeline which runs Playwright tests on schedule (Mon-Fri every 4 hours) against the Test (Dev env is less stable) env.
In addition, fixed existing schedule for the `nightly` build-deploy pipeline because scheduled Workflow will stop working at the end of year. Also removed boilerplate code by using matrix. See https://circleci.com/docs/workflows/#scheduling-a-workflow

Project Settings => Triggers:
https://app.circleci.com/settings/project/github/broadinstitute/ddp-angular/triggers?return-to=https%3A%2F%2Fapp.circleci.com%2Fpipelines%2Fgithub%2Fbroadinstitute%2Fddp-angular

Screen Shot 2022-11-04 at 12 32 08 PM

### CODE REVIEW

File: .circleci/config.yml
Comment (aweng98) [inline .circleci/config.yml:661]: Don't need this anymore, used for troubleshooting playwright-test job.

File: .circleci/config.yml
Comment (aweng98) [inline .circleci/config.yml]: Workflow schedule will be phased out by CircleCI at the end of year. See note https://circleci.com/docs/workflows/#scheduling-a-workflow

New pipeline trigger created. See screenshots in PR description.

File: .circleci/config.yml
Comment (aweng98) [inline .circleci/config.yml]: Renamed pipeline name from `nightly` to `nightly-deploy` for distinction from nightly playwright test pipeline.

File: .circleci/config.yml
Comment (aweng98) [inline .circleci/config.yml]: Using `matrix`, removed duplicated boilerplate code.
See https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/20893/workflows/3544dcbf-8cfe-4343-bba5-6526d2e7867a

**Review by Rinchan20 — DISMISSED** [review]
looks good imho - good call on trying to limit load

File: .circleci/config.yml
Comment (ungwudik) [inline .circleci/config.yml]: Nit: should it still be called `nightly`?

File: .circleci/config.yml
Comment (ungwudik) [inline .circleci/config.yml]: Nit: is this still up-to-date?

**Review by ungwudik — COMMENTED** [review]
Playwright schedule looks good to me. The refactoring/improvement is a welcome addition, but I'm not familiar with it, so I'll defer to another person to review that.

File: .circleci/config.yml
Comment (aweng98) [reply .circleci/config.yml]: No, I'll fix it. Good catch.

File: .circleci/config.yml
Comment (aweng98) [reply .circleci/config.yml]: I'm looking for suggestion for a better workflow name. It shouldn't be `hourly`.

**Review by alextanlan — DISMISSED** [review]

**Comment by alextanlan** [comment]
Just wondering what is the reason to run e2e tests 6 times per day? 
Specially if we update test env not so often.
And it is not exactly "nightly" run in this case.

File: .circleci/config.yml
Comment (MocanaAtBroad) [inline .circleci/config.yml]: Since you are touching this, let's just get rid of this nightly build-deploy. It is useless.

File: .circleci/config.yml
Comment (MocanaAtBroad) [reply .circleci/config.yml]: "scheduled"?

File: .circleci/config.yml
Comment (aweng98) [reply .circleci/config.yml]: Sounds good. Can I do it in a follow-up PR? Because new commits will remove approval.

File: .circleci/config.yml
Comment (aweng98) [reply .circleci/config.yml]: Can make this change in a follow-up PR as well.

**Comment by aweng98** [comment]
> Just wondering what is the reason to run e2e tests 6 times per day? Specially if we update test env not so often. And it is not exactly "nightly" run in this case.

Several folks wanted to see tests run more often to gather data about test flakes.

File: .circleci/config.yml
Comment (MocanaAtBroad) [reply .circleci/config.yml]: For a deletion I can approve and get you another approval quickly.

Comment (aweng98) [reply .circleci/config.yml]: Removed old nightly workflow.

File: .circleci/config.yml
Comment (aweng98) [reply .circleci/config.yml]: Renamed to workflow name to `scheduled-playwright-test`.

File: .circleci/config.yml
Comment (aweng98) [reply .circleci/config.yml]: Fixed comment.

**Review by MocanaAtBroad — APPROVED** [review]

**Review by alextanlan — APPROVED** [review]

**Pull request merged**
